prombench/manifests/prombench/benchmark/6_loadgen.yaml

```diff
@@ -48,6 +48,25 @@ data:
         - expr: histogram_quantile(0.99, sum by(path, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(path, method, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(instance, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
+      - name: binary_arithmetic_operators
+        interval: 15s
+        type: instant
+        queries:
+        - expr: sum(node_memory_MemAvailable_bytes) / sum(node_memory_MemTotal_bytes)
```

Comment:
I checked [a running Prombench](http://prombench.prometheus.io/14920/prometheus-release/graph?g0.expr=count(node_memory_MemAvailable_bytes)&g0.tab=1&g0.display_mode=lines&g0.show_exemplars=0&g0.range_input=1h), and there are only 3 series like this; perhaps not great for a benchmark.

Reply:
@bboreham Hi Sir, I hope you are doing well. First of all, sorry for the late reply. I'm also really glad that you liked my #743 PR. 

So, now that I'm focusing on this, here are some of the suggested changes from my side. PTAL once so I can create a commit and merge it.

```
- name: binary_arithmetic_operators
  interval: 15s
  type: instant
  queries:
  - expr: sum(rate(codelab_api_requests_total[5m])) / count(codelab_api_requests_total)
  - expr: rate(codelab_api_request_duration_seconds_count[5m]) * 60
  - expr: codelab_api_request_duration_seconds_sum / codelab_api_request_duration_seconds_count
```

As suggested by you, I have made changes, PTAL.